crates/rolldown_error/Cargo.toml

```diff
@@ -21,10 +21,10 @@ anyhow  = { workspace = true }
 arcstr  = { workspace = true }
 ariadne = { workspace = true }
 heck    = { workspace = true }
-# memchr        = { workspace = true }
 napi          = { workspace = true, optional = true }
 oxc           = { workspace = true }
 oxc_resolver  = { workspace = true }
 oxc_sourcemap = { workspace = true }
 rustc-hash    = { workspace = true }
 sugar_path    = { workspace = true }
+ropey        = { workspace = true }
```

Comment:
Add a new dependency here. It is used to convert the byte index to the char index. This is fine because it was previously indirectly dependent on rolldown, so it will not increase binary output.